core/jvm/src/test/scala/scalaz/zio/PromiseSpec.scala

```diff
@@ -52,7 +52,7 @@ class PromiseSpec extends Specification with RTS {
     unsafeRun(
       for {
         p <- Promise.make[String, Int]
-        s <- p.done[String](ExitResult.Failed("error in e4"))
+        s <- p.done[String](ExitResult.Failed("error in e4", Nil))
```

Comment:
Something to think when reviewing the design of `ExitResult` is that errors are lost when the action fails within `attempt`, as only the `E` is produced in the value channel.

Reply:
I haven't touched `attempt`, it would require changing it to something like `IO[Nothing, Either[(E, List[Throwable]), A]]` 😱